packaging_automation/packaging_warning_handler.py

```diff
@@ -40,33 +40,38 @@ def validate_output(output: str, ignore_file_path: str, package_type: PackageTyp
 
 
 def filter_warning_lines(output_lines: List[str], package_type: PackageType) -> Tuple[List[str], List[str]]:
+    rpm_warning_summary = r"\d+ packages and \d+ specfiles checked; \d+ errors, \d+ warnings."
+    rpm_lintian_starter = 'Executing "/usr/bin/rpmlint -f /rpmlintrc'
+    debian_lintian_starter = "Now running lintian"
+    lintian_warning_error_pattern = r".*: [W|E]: .*"
+
     base_warning_lines = []
     package_specific_warning_lines = []
     is_deb_warning_line = False
     is_rpm_warning_line = False
     for output_line in output_lines:
+
         if package_type == PackageType.deb:
-            if "warning" in output_line.lower() or is_deb_warning_line:
-                if "Warnings generated by release docker build" in output_line:
-                    is_deb_warning_line = True
-                    continue
+            if debian_lintian_starter in output_line:
```

Comment:
Here there were some error preventing debian warnings to be handled. First lintian starter line to be checked to handle W E like errors (like RPM below)